client/app/index.ts

```diff
@@ -1,7 +1,7 @@
 import type { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
 import axios from "axios";
 
-const localHost = "http://localhost:3010/api";
+const localHost = "http://localhost:3000/api";
```

Comment:
Так тоже должно работать:
```suggestion
const localHost = "/api";
```